js/build.js

```diff
@@ -133,6 +134,25 @@
         })
       }
 
+      /**
+       * A function that searches for a personal style of the widget
+       *
+       * @returns {object/null} returns a color object or false if there is no personal color used for this widget
+       */
+      function getPersonalColors() {
```

Comment:
Use the word custom/customize, not personal.

```suggestion
      function getCustomColors() {
```